br/tests/lightning_duplicate_resolution_error/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright 2021 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+check_cluster_version 5 2 0 'duplicate detection' || exit 0
+
+mydir=$(dirname "${BASH_SOURCE[0]}")
+
+run_sql 'DROP TABLE IF EXISTS dup_resolve.a'
+run_sql 'DROP TABLE IF EXISTS lightning_task_info.conflict_error_v2'
+
+! run_lightning --backend local --config "${mydir}/config.toml"
+[ $? -eq 0 ]
+
+tail -n 10 $TEST_DIR/lightning.log | grep "ERROR" | tail -n 1 | grep -Fq "[Lightning:Restore:ErrFoundDuplicateKey]found duplicate key"
```

Comment:
can you also check the output key is human readable? also ask PM what's the expected format.

Reply:
Currently the output looks like this
```
[error="[Lightning:Restore:ErrFoundDuplicateKey]found duplicate key 't\ufffd\u0000\u0000\u0000\u0000\u0000\u0000l_r\ufffd\u0000\u0000\u0000\u0000\u0000\u0000\u0001', value '\ufffd\u0000\u0002\u0000\u0000\u0000\u0002\u0003\u0001\u0000\u0006\u0000\u00061.csv'"]
```

Reply:
"error" is only internally used before, it does not consider human-readability.

Reply:
DDL uses "error" option before, and has extra work to extract the data, like

```
			if common.ErrFoundDuplicateKeys.Equal(err) {
				err = convertToKeyExistsErr(err, indexInfo, tbl.Meta())
			}
```

We can mimic it or we directly read other fields from the conflict error table and display them to user. Please be careful do not break DDL's behaviour.

Reply:
Will address in next PR.